pkg/manager/member/pd_member_manager_test.go

```diff
@@ -1191,6 +1191,76 @@ func TestGetNewPDSetForTidbCluster(t *testing.T) {
 				g.Expect(hasTLSVolMount(sts)).To(BeTrue())
 			},
 		},
+		{
+			name: "tidbcluster with failureMember nonDeleted",
+			tc: v1alpha1.TidbCluster{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "tls-nightly",
+					Namespace: "ns",
+				},
+				Spec: v1alpha1.TidbClusterSpec{
+					PD: v1alpha1.PDSpec{
+						ComponentSpec: v1alpha1.ComponentSpec{
+							Image: "pingcap/pd:nightly",
+						},
+						Replicas: 3,
+					},
+					TiDB: v1alpha1.TiDBSpec{
+						TLSClient: &v1alpha1.TiDBTLSClient{
+							Enabled: true,
+						},
+					},
```

Comment:
can this `TiDB` field be removed? we can make our tests as short as possible

Reply:
updated